src/lib/index.d.ts

```diff
@@ -18,8 +18,9 @@ export type WebsocketAdapterConfig = {
     port?: number,
   },
   client?: {
+    query?: any
     authentication?: {
```

Comment:
```suggestion
   headers?: any
```

Instead of adding authentication should we just allow header and query config, with headers the user is able to set any type of authentication he desires?

Reply:
But the user can already do that once you merge [this PR of yours](https://github.com/asyncapi/bindings/pull/144/files), right? We only saw the need for authentication because it usually contains secrets.

Reply:
Yeah, just a suggestion, currently we have an `authentication` config and we are creating a header `Authentication: bearer TOKEN`, and it works but maybe we just allow headers instead of authentication, and then we have a broader control, but we only have the need for authentication. We can always change it in the future if there is a need.

Reply:
We'd have broader control at Glee but we'd be breaking the purpose of the AsyncAPI specification. If the `headers` values are in the AsyncAPI document, other tools will be able to see them. If they're in the Glee config, only Glee will be able to see them. You see the problem?

For instance, in #261 we want to use Generator to generate docs automatically. If the info is not in the AsyncAPI file, it will not end up in the documentation. And the same will happen for all the tools that the user might want to apply to their AsyncAPI file.